detect_secrets/core/audit.py

```diff
@@ -19,18 +20,30 @@ class SecretNotFoundOnSpecifiedLineError(Exception):
     pass
 
 
+def _clean_baseline_of_nonexistent_files(baseline):
```

Comment:
`_remove_nonexistent_files_from_baseline`? And it should return `True` if modified -- just so it's less negatives, so we don't get a case where:

```
if not baseline_wasnt_modified:
    do_something()
```

nit: let's throw this below `audit_baseline` function declaration, so it reads from top-down.

Reply:
Hmm, I think I originally made it that but I changed it to `wasnt_modified` was to avoid the `and not baseline_was_modified`, which I read as slightly more confusing. This may be a weird preference of mine though, I'll change it ;)

Reply:
Hmm `and not files_removed:` looks nice 👍